lib/matplotlib/figure.py

```diff
@@ -32,7 +32,7 @@
 
 from matplotlib.cbook import Stack, iterable
 
-from matplotlib import _image
+from matplotlib import image as mimage
```

Comment:
" lib/matplotlib/tests/baseline_images/test_backend_svg/noscale.png"
doesn't look right to me.

On Wed, Jan 6, 2016 at 2:11 PM, Thomas A Caswell <notifications@github.com>
wrote:

> In lib/matplotlib/figure.py
> <https://github.com/matplotlib/matplotlib/pull/5718#discussion_r48995898>:
>
> > @@ -32,7 +32,7 @@
> >
> >  from matplotlib.cbook import Stack, iterable
> >
> > -from matplotlib import _image
> > +from matplotlib import image as mimage
>
> I sometimes have thoughts that __all__ = [] should be added to all python
> files [image: :smiling_imp:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/matplotlib/matplotlib/pull/5718/files#r48995898>.
>